src/extension.ts

```diff
@@ -253,6 +261,9 @@ class ResMon {
             // Get the display of the requested resources
             let pendingUpdates: Promise<string | null>[] = resources.map(resource => resource.getResourceDisplay());
 
+            //change color
```

Comment:
Nit comment style:
```
// Change color
```

Reply:
I would probably just remove this comment...
Does the default color work as expected for both the light and dark theme?